docs/destinations/PDODestination.md

```diff
@@ -44,7 +44,7 @@ $pdoDestination->ignoreIntegrityConstraintViolations();
 
 # Transactions
 
-By default PDO transactions are enabled. This helps speed up the migration process by keeping changes in memory until they are committed to disk. If you do not want to have transactions or is not supported you can easily disable this, as shown below.
+By default, database transactions are enabled. This helps speed up the migration process by keeping changes in memory until they are committed to disk. If you do not want to use transactions or they not supported by your database engine, you can easily disable them, as shown below.
```

Comment:
> By default, database transactions are enabled.

Surely this should be specific as you might have a MYSQLi destination at some point?

Reply:
It's specifically in the PDO Destination documentation, so I figured that was unnecessary.

Reply:
Ah true, inside a specific file

Reply:
:+1: